mri/operators/fourier/utils.py

```diff
@@ -216,6 +217,11 @@ def gridded_inverse_fourier_transform_stack(kspace_data_sorted,
     This function calculates the gridded Inverse fourier transform
     from Interpolated non-Cartesian data into a cartesian grid. However,
     the IFFT is done similar to Stacked Fourier transform.
+    We expect the kspace data to be limited to a grid on z, we calculate
+    the inverse fourier transform by-
+    1) Grid data in each plane (for all points in a plane)
+    2) Interpolate data along z, if we have undersampled data along z
+    3) Apply an IFFT on the 3D data that was gridded.
```

Comment:
gridded then interpolated.

Reply:
Yes, that is what I have written right? `1)` Is gridding on each plane, `2)` is the actual interpolation in z.
Do you want me to explicitly mention this again in point 2?

Reply:
yes yes, I just think it's better reminded in point 3